ietf/meeting/tests_js.py

```diff
@@ -475,6 +475,14 @@ def test_past_swap_days_buttons(self):
         # option to swap. If we used the first or last day, a fencepost error in
         # disabling options by date might be hidden.
         clicked_index = 1
+        # scroll so the button we want to click is just below the navbar, otherwise it may
+        # fall beneath the sessions panel
+        navbar = self.driver.find_element_by_class_name('navbar')
+        self.driver.execute_script(
+            'window.scrollBy({top: %s, behavior: "instant"})' % (
+                    future_swap_days_buttons[1].location['y'] - navbar.size['height']
+            )
+        )
```

Comment:
Does `self.scroll_to_element` not work?

Reply:
No, unfortunately. I think that only scrolls until the element is in the viewport. The session panel is still floating over it, intercepting clicks.

(Sorry to repeat, but this showed up as a top level comment rather than a reply to your question)